unicodetools/org/unicode/propstest/UnicodePropertyPartition.java

```diff
@@ -0,0 +1,90 @@
+package org.unicode.propstest;
+
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.Map.Entry;
+
+import org.unicode.cldr.util.Counter;
+import org.unicode.props.IndexUnicodeProperties;
+import org.unicode.props.PropertyStatus;
+import org.unicode.props.PropertyStatus.PropertyOrigin;
+import org.unicode.props.PropertyType;
+import org.unicode.props.UcdProperty;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Multimap;
+import com.google.common.collect.Ordering;
+import com.google.common.collect.TreeMultimap;
+import com.ibm.icu.dev.util.UnicodeMap;
+import com.ibm.icu.text.UnicodeSet;
+
+public class UnicodePropertyPartition {
```

Comment:
Please add a comment for what this does, and maybe a little about how.

Reply:
Actually, I shouldn't have included this in the PR. Removing.